docs/csharp/programming-guide/concepts/covariance-contravariance/using-variance-in-delegates.md

```diff
@@ -44,8 +44,23 @@ class Program
   
 ## Example 2: Contravariance  
   
-### Description  
- This example demonstrates how delegates can be used with methods that have parameters of a type that are base types of the delegate signature parameter type. With contravariance, you can use one event handler instead of separate handlers. For example, you can create an event handler that accepts an `EventArgs` input parameter and use it with a `Button.MouseClick` event that sends a `MouseEventArgs` type as a parameter, and also with a `TextBox.KeyDown` event that sends a `KeyEventArgs` parameter.  
+### Description
+
+This example demonstrates how delegates can be used with methods that have parameters whose types are base types of the delegate signature parameter type. With contravariance, you can use one event handler instead of separate handlers. The following example makes use of two delegates:
+
+- A <xref:System.Windows.Forms.KeyEventHandler> delegate that defines the signature of the [Button.KeyDown](xref:System.Windows.Forms.Control.KeyDown) event. Its signature is:
+
+   ```csharp
+   Public Delegate Sub KeyEventHandler(object sender, KeyEventArgs e)
+   ```
+
+- A <xref:System.Windows.Forms.MouseEventHandler> delegate that defines the signature of the [Button.MouseClick](xref:System.Windows.Forms.Control.MouseDown) event. Its signature is:
+
+   ```csharp
+   Public Delegate Sub MouseEventHandler(object sender, MouseEventArgs e)
+   ```
```

Comment:
Both signatures look like a mix of VB and C#, they're not valid C#: `Public Delegate Sub` should be `public delegate void`.

Reply:
How embarrassing. Thanks for noticing it, @svick. I'll fix it, and also include the VB version of this document in the PR.